### DISCUSSION TITLE
Allow escaping from within template

### DISCUSSION THREAD

**Discussion by acalvino4** [ExpressionEngine/ExpressionEngine#2927] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Currently there is no way to escape content in a template so that the parser outputs it as-is. That a mature templating engine should have this capability is a no-brainer to me. The other big names out there in templating have it:
https://laravel.com/docs/8.x/blade#the-at-verbatim-directive
https://twig.symfony.com/doc/3.x/tags/autoescape.html

**Describe the solution you'd like**
There should be a tag pair that designates everything in between get's output as-is, perhaps
`{escape}content-including-curly-braces{/escape}`

**Describe alternatives you've considered**
The alternatives are to use a global template variable, or channel for storing this kind of content. But this is of course a ridiculous way to handle that in the modern development world.

**Teachability, Documentation, Adoption, Migration Strategy**
We should easily be able to add a page to the documentation outlining this new feature once the exact format is decided. The main decision to make would be where exactly in the template parse order this occurs, but it would probably be somewhere near the beginning.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
ExpressionEngine does have `protect_javascript` property of `Template` model (which I belive is intended to work similarly to what you envision here), but it is not available for manipulations in CP controller with latest EE 5.x and 6.x.

@robinsowell do you have any insight of when and why it has been removed?

**Comment by robinsowell** [comment]
@intoeetive no, I'm remembering when it was made a per template setting, but nothing about it being removed.  I skimmed the changelogs and don't see a reference either.  Though I'm also not sure how it would have been accidentally dropped.

**Comment by intoeetive** [comment]
Looking into this again, it looks like `protect_javascript` only 
> Prevents advanced conditional parser from processing anything in <script> tags

And this request is allowing something to be exluded from parsing completely. I think this is valid, and would be a nice feature.

For the time being though, people can use `Static` templates, which are not being parsed. So same thing, but on template level.
